### DISCUSSION TITLE
battery capacity

### DISCUSSION THREAD

**Discussion by R3Cardoso** [wlcrs/huawei_solar#468] [Setting up the integration] [answered]
I got this integration working for some months, but I think I just noticed I got my battery capacity wrong configured, but I need to be sure.
I have two moduls from the 5kw capacity, that gives me a total battery capacity of 10kw.
Shoud I change this values on this fields:
<img width="343" alt="image" src="https://github.com/wlcrs/huawei_solar/assets/119068070/71b1f03c-96ab-48c5-8dda-f7dbf19adac0">
What should be on Battery Maximum charging power? 10000?
What should be on Battery Maximum discharging power?

**Comment by Roving-Ronin** [comment, marked as answer]
If you have 2 x 5kWh modules, you have 10kWh capacity.
Maximum charging and discharging capacity for LUNA 10kWh / 15kWh configuration is 5kW  ( 5,000W )
See:
![image](https://github.com/wlcrs/huawei_solar/assets/108674933/1778cb65-7a9c-431d-96fb-e6b7a75c61da)

**Reply by R3Cardoso** [reply]
Yes thats the module I have, luna 2000.
Ty for the information.
I also had noticed I can't type a higher value than 5000w

**Reply by Roving-Ronin** [reply]
As above, you have 10kWh CAPACITY, with 5000W Max Output (i.e. Charge / Discharge).
